docs/content/kubernetes-primer.md

```diff
@@ -95,7 +95,7 @@ In OPA, `input` is a reserved, global variable whose value is the  Kubernetes Ad
 AdmissionReview objects have many fields.  The rule above uses `input.request.kind`, which includes the usual group/version/kind information.  The rule also uses `input.request.object`, which is the YAML that the user provided to `kubectl` (augmented with defaults, timestamps, etc.).  The full `input` object is 50+ lines of YAML, so below we show just the relevant parts.
 
 ```yaml
-apiVersion: admission.k8s.io/v1beta1
+apiVersion: admission.k8s.io/v1
```

Comment:
I was wondering about the compatibility with k8s, but it seems like 

- `v1beta1` was deprecated in v1.16 in favor of admission.k8s.io/v1
- k8s 1.15 went out of support on 6 May 2020a